src/collective/upload/profiles/default/jsregistry.xml

```diff
@@ -54,7 +54,7 @@
       expression="" inline="False" authenticated="True" />
 
   <!-- Bootstrap JS -->
-  <javascript id="++resource++collective.upload/vendor/bootstrap.min.js"
+  <javascript id="++resource++collective.js.bootstrap/js/bootstrap.min.js"
```

Comment:
@rodfersou installing this resource is a job of collective.js.bootstrap; check collective.cover for instance.